main/src/main/java/org/mobilitydata/gtfsvalidator/validator/ShapeIncreasingDistanceValidator.java

```diff
@@ -28,7 +28,9 @@
 /**
  * Validates that shape_dist_traveled along a shape in "shapes.txt" are not decreasing.
  *
- * <p>Generated notice: {@link DecreasingOrEqualShapeDistanceNotice}.
+ * <p>Generated notice: {@link DecreasingOrEqualShapeDistanceNotice}. *
+ *
+ * <p>Severity: {@code SeverityLevel.ERROR}
```

Comment:
In the other `*Validator` classes we don't include the SeverityLevel, so I'd remove this here.

Also looks like there is an extra `*` at the end of the line above.